code/obj/item/gun/ammo.dm

```diff
@@ -1294,15 +1294,10 @@ ABSTRACT_TYPE(/obj/item/ammo/bullets/pipeshot)
 	desc = "A self-contained radioisotope power cell that very slowly recharges an internal capacitor. Holds 40PU."
 	recharge_rate = 2.5 // cogwerks: raised from 1.0 because radbows were terrible!!!!!
 
-/obj/item/ammo/power_cell/self_charging/disruptor
-	name = "Power Cell - Disruptor Charger"
-	desc = "A self-contained radioisotope power cell that slowly recharges an internal capacitor. Holds 100PU."
-	icon = 'icons/obj/items/ammo.dmi'
-	icon_state = "recharger_cell"
-	m_amt = 18000
-	g_amt = 38000
-	charge = 100
-	max_charge = 100
```

Comment:
You deleted the parent disruptor cell. Assuming that's not intentional?

Reply:
It is not! Remnant of a change before I settled on this one, fixed now